.github/workflows/tests.yml

```diff
@@ -0,0 +1,14 @@
+name: Run Tests
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6'
+    - name: RSpec
```

Comment:
This name seems rather specific given the step does more than this (bundle and generic rake) where rspec is just one of the things that happens